provisioner/terraform/executor.go

```diff
@@ -609,12 +676,25 @@ func convertTerraformLogLevel(logLevel string, sink logSink) proto.LogLevel {
 }
 
 type terraformProvisionLog struct {
-	Level   string `json:"@level"`
-	Message string `json:"@message"`
+	Level     string                    `json:"@level"`
```

Comment:
I expanded this type to unmarshal additional fields from the existing JSON logs.